src/program.ts

```diff
@@ -193,6 +198,50 @@ program
         launch(program, options)
     })
 
+program
+    .command('playbook_init')
```

Comment:
Use a subcommand approach not a _ approach pls. check docs for commander for reference.